subprojects/docs/src/docs/userguide/extending-gradle/custom_tasks.adoc

```diff
@@ -139,17 +139,6 @@ This can be particularly useful for a transformer task that converts input files
 
 If you'd like to optimize your build so that only out-of-date input files are processed, you can do so with an _incremental task_.
```

Comment:
```suggestion
You can optimize your build to only process out-of-date input files with an _incremental task_. To learn more, refer to the link:{groovyDslPath}/org.gradle.work.InputChanges.html[InputChanges] API documentation.
```

Reply:
Note that after that it's described how you can use InputChanges and API link is in second sentence of the next paragraph. Check: https://docs.gradle.org/current/userguide/custom_tasks.html#incremental_tasks. 

```
For a task to process inputs incrementally, that task must contain an _incremental task action_.
This is a task action method that has a single link:{groovyDslPath}/org.gradle.work.InputChanges.html[InputChanges] parameter.
```

Reply:
Thanks for pointing that out. Feel free to disregard this then.